tests/test_default_router.py

```diff
@@ -159,6 +159,10 @@ def test_invalid_field_name(self, template):
             ValueError,
             self.router.add_route, template, {}, ResourceWithId(-1))
 
+    def test_single_character_field_name(self):
+        self.router.add_route(
+            '/search?q={q}', {}, ResourceWithId(22))
```

Comment:
Perhaps I'm just overlooking something, but I think we need a assertion here to verify this works as expected.